package.json

```diff
@@ -53,15 +53,19 @@
   },
   "homepage": "https://github.com/benmosher/eslint-plugin-import",
   "devDependencies": {
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.7",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/register": "^7.8.6",
     "@eslint/import-test-order-redirect-scoped": "file:./tests/files/order-redirect-scoped",
     "@test-scope/some-module": "file:./tests/files/symlinked-module",
+    "@types/chai": "^4.2.11",
```

Comment:
This project isn’t in typescript, so these types aren’t useful.

Reply:
@ljharb ok, I removed them. :+1:

Usually, I still like to have them still as it improves the IDE autocompletion. But there are ways around it.